comms/Cargo.toml

```diff
@@ -39,6 +39,7 @@ tokio = {version="~0.2.19", features=["blocking", "time", "tcp", "dns", "sync",
 tokio-util = {version="0.2.0", features=["codec"]}
 tower= "0.3.1"
 yamux = "=0.4.7"
+itertools = "0.9.0"
```

Comment:
You can remove this dependency (see comment in comms/src/peer_manager/node_id.rs)